features/selectusers/api/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("io.element.android-compose-library")
+}
+
+android {
+    namespace = "io.element.android.features.selectusers.api"
```

Comment:
not a big deal, but maybe `userpicker` (or `userspicker`) would be a better name for this module.

Reply:
Done https://github.com/vector-im/element-x-android/pull/223/commits/dfbe15b5be08ed8ebdf49ec688f3e3dfa07e5f18 and https://github.com/vector-im/element-x-android/pull/223/commits/690d8efb4036675382aba2275fb2d48545ba3708